addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1928,11 +1937,22 @@ var SnippetsMenu = Widget.extend({
     _disableUndroppableSnippets: function () {
         var self = this;
         var cache = {};
+
+        const allExcludeDropInOptions = self.templateOptions.filter(option => {
+            return option['exclude-drop-in'];
+        });
```

Comment:
See I don't understand the point of this code, if the exclusion was directly part of the `all` (etc) methods, then this code would probably not have to be changed right?

Basically writting: `data-drop-in="A, B" data-exclude-drop-in="C"` is just a shortcut for `data-drop-in="A:not(C), B:not(C)"` so it really should not impact much code other than the computation of the drop-in function.